directives/questionnaire.py

```diff
@@ -113,6 +114,14 @@ def run(self):
         }
         self.set_assistant_permissions(data)
 
+        points_set_in_arguments = False
+        if len(self.arguments) == 2 and difficulty != self.arguments[1]:
+            points_set_in_arguments = True
+
+        if points_set_in_arguments and env.aplus_quiz_total_points != points:
+                raise SphinxError('The points of the questions in the questionnaire must add up to the totalpoints of the questionnaire!')
+        data['max_points']=env.aplus_quiz_total_points
```

Comment:
Add spaces around the `=` assignment operator.

Reply:
It looks like you could remove the `max_points` key from `data` when the dict is first initialized since the key is always set here again.